### ISSUE TITLE
Grid is not on PC Game Pass

### ISSUE THREAD

**Issue by brande77** [alike03/SubscriptionInfo#20]
https://store.steampowered.com/app/703860/GRID/

It's listed with PC Game Pass tag, but it's Xbox console via Game Pass' included EA Play. It not available on PC at all with Game Pass or EA Play.

**Comment by alike03** [comment]
It is, some PC Game Pass games included via EA Play are only available on the EA Desktop app and are not listed in the Xbox App. 
So you have to install that and start it via EA Desktop after connecting it with your Xbox account.

**Comment by brande77** [comment]
No, it appears to not be. You can not even get it directly from EA Play desktop client. The only "Grid" available is "Grid Legends".
https://store.steampowered.com/app/1307710/GRID_Legends/
Which is already correctly labeled with the script.

There is no "Grid" available with Game Pass or EA Play.
https://i.imgur.com/kzeP25X.jpeg

**Comment by alike03** [comment]
Sorry, I got it mixed up with Grid Legends. 
Technically, it is on EA Play, but only if you have EA Play via Steam.
After Codemasters was acquired by EA, they began including all of their previous games in EA Play.
I looked a bit into it. EA stated, that they are working on bringing the games to EA Desktop, but after a year still nothing. 
I wouldn't get my hopes up.
I will look more into this and probably exclude non EA Desktop games, since Steam does show "EA Play Steam" games.
Thanks for bringing this to my attention.
